src/ecs/commands/runCommandInContainer.ts

```diff
@@ -187,3 +146,58 @@ async function verifyCliAndPlugin(window: Window): Promise<boolean> {
     }
     return true
 }
+
+async function isDeploymentInProgress(node: EcsContainerNode): Promise<boolean> {
+    const deployments = (await node.ecs.describeServices(node.clusterArn, [node.serviceName]))[0].deployments
+    if (deployments) {
+        for (let i = 0; i < deployments.length; i++) {
+            if (deployments[i].status === 'ACTIVE' || deployments[i].rolloutState === 'IN_PROGRESS') {
+                return true 
```

Comment:
return the status so it can be surfaced in the message. Precise messages are more helpful.